### PR TITLE
Added check for Datetime in exif of images

### PR DESCRIPTION
**Pull request by Manusreekumar**
In certain new incoming images, the Date time value wast not present in exif, leading to key error. Hence an additional check is added to ensure pre processing runs even in the absence of date time

### CODE REVIEW

**Comment by Manusreekumar** [comment]
@gsganden There seems to be an issue while build.  "The command "python setup.py install" failed and exited with 1 during ." error is coming.  Would you be able to check ?

**Comment by gsganden** [comment]
I've seen this PR, will get to it when I can. FYI I'm doing all of my development in the `reorganize` branch, which pretty much rewrites the whole repo from the ground up. Small fixes to `master` are a low priority for me, and I would recommend working off of `reorganize`.

**Comment by Manusreekumar** [comment]
Sounds great. I am currently working on improving model capabilities by enabling the model to learn specific animal images. I would like to learn about the changes you are planning to make and also run certain ideas that I am planning to work by you.

**Comment by gsganden** [comment]
Cool. That branch has nearly complete code for retrieving the 2016/2017 data and processing it. It's slow going because the dataset is largely enough to make each uploading, downloading, and processing step a hassle.

I'm also working to improve the human model and create models for other species as well. Then I want to pull in the 2018 data.

**Comment by gsganden** [comment]
Good change. I would prefer this slightly more compact implementation:

```
timestamp = exif.get('DateTime')
if timestamp is not None:
    timestamp = datetime.strptime(timestamp, '%Y:%m:%d %H:%M:%S')
```

File: autofocus/preprocess_images.py [hunk 1]
```diff
@@ -136,8 +136,11 @@ def get_timestamp(exif: dict) -> datetime:
     Args:
         exif: dictionary of Exif metadata that includes "DateTime"
     """
-    timestamp = exif['DateTime']
-    timestamp = datetime.strptime(timestamp, '%Y:%m:%d %H:%M:%S')
+    if 'DateTime' in exif:
```
Comment (gsganden) [inline autofocus/preprocess_images.py]: Good change. I would prefer this slightly more compact implementation:

```
timestamp = exif.get('DateTime')
if timestamp is not None:
    timestamp = datetime.strptime(timestamp, '%Y:%m:%d %H:%M:%S')
```

**Comment by gsganden** [comment]
Closing this PR because it will soon be obsolete.

**Pull request closed without merging**